af_ktls.c

```diff
@@ -984,10 +988,14 @@ static inline ssize_t tls_peek_data(struct tls_sock *tsk, unsigned flags)
 			goto peek_failure;
 		}
 
+		if (peeked_size < KTLS_MIN_RECORD_SIZE(tsk)) {
+			ret = -EBADF;
```

Comment:
Good point, thanks. I don't know if no error is OK here. What about EAGAIN for user space 
```if peeked_size < KTLS_MIN_RECORD_SIZE(tsk)``` 
and EBADF for
```if peeked_size > KTLS_MIN_RECORD_SIZE(tsk)``` ?